group_vars/tigerdata/common.yml

```diff
@@ -0,0 +1,47 @@
+postgresql_is_local: false
+postgres_version: 13
+postgres_admin_password: '{{ vault_postgres_admin_password }}'
+pg_hba_contype: "host"
+pg_hba_postgresql_user: "all"
+pg_hba_postgresql_database: "all"
+pg_hba_method: "md5"
+pg_hba_source: "{{ ansible_host }}/32"
+postgres_admin_user: "postgres"
+application_db_name: '{{ app_db_name }}'
+application_dbuser_name: '{{ app_db_user }}'
+application_dbuser_password: '{{ app_db_password }}'
+application_dbuser_role_attr_flags: 'CREATEDB'
+app_host_name: '{{ passenger_server_name }}'
+application_host_protocol: 'https'
+running_on_server: true
+tigerdata_honeybadger_key: '{{ vault_honeybadger_api_key }}'
+bundler_version: '2.3.26'
+desired_nodejs_version: 'v16.19.0'
```

Comment:
tiger-data-app tool-versions has
```
nodejs 18.14.0
```
Would it be safer for these to match?

Reply:
Unfortunately we do not have 18.14.0, but I updated it to18.13.0